ooni/scripts/ooniprobe_agent.py

```diff
@@ -121,6 +121,16 @@ def get_running_pidfile():
             return pidfile
     raise NotRunning
 
+def is_stale_pidfile(pidfile):
+    if os.path.exists(pidfile):
+        with open(pidfile) as fd:
```

Comment:
I would wrap this inside of a try except to avoid races between the `exists` check and the opening of the file.